### ISSUE TITLE
iTerm 3.0.3 Can’t make "window-1" into type integer. (-1700)

### ISSUE THREAD

**Issue by ptim** [TomAnthony/itermocil#50]
Heh, saw the change in the iterm update changelog, but couldn't resist :)

The error is: 

`140:176: execution error: iTerm got an error: Can’t make "window-1" into type integer. (-1700)`

And the changelog message was something like (couldn't find a link to it, sorry):

>Return a real window.ID (integer) rather than a string

Thanks, really appreciate itermocil :)

**Comment by binderclip** [comment]
@ptim Glad to know this has been fixed by iTerm 3.0.4. 😄  

```
- PLEASE NOTE: If you had trouble upgrading before, it should be fixed now! -

iTerm 3.0.4
This release has only one bug fix.

- Fixes how windows are referred to by Applescript, which 3.0.3 broke.
```

**Comment by ptim** [comment]
sweet, all good!
